src/components/AdminPane/Manage/ViewChallengeTasks/ViewChallengeTasks.jsx

```diff
@@ -153,7 +153,7 @@ export class ViewChallengeTasks extends Component {
   // bounding box overlays on the map.
   findPriorityBounds = (challenge) => {
     const parseBoundsRule = (rule, priorityLevel, priorityBounds) => {
-      if (rule.rules) {
+      if (rule?.rules) {
```

Comment:
This seems suspect: if `rule` can be nullish, then the `if (rule.type === "bounds")` line should also guard for this, otherwise it'll crash with `undefined is not an object`.